doc/classes/CanvasItem.xml

```diff
@@ -405,6 +405,18 @@
 				Returns the [RID] of the [World2D] canvas where this item is in.
 			</description>
 		</method>
+		<method name="get_canvas_layer" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns the layer number of the [CanvasLayer] where this node is in.
```

Comment:
This description should also mention the special case that the CanvasItem can be in the viewports default canvas layer.

Reply:
> This description should also mention what happens, when there is no `CanvasLayer` or alternatively replace "[CanvasLayer]" by "canvas layer".

Thank you! I've already changed it and mentioned that `0` will be returned if there's no `CanvasLayer`.